sql/migrations/20210624194440_world.sql

```diff
@@ -0,0 +1,25 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20210624194440');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20210624194440');
+-- Add your query below.
+
+
+-- Correct Barbs Aura Passive for Mudrock Spikeshell 4397 (credit cmangos)
+UPDATE `creature_template` SET `auras` = '9464' WHERE `entry` = 4397;
+
+-- Add Missing Loot Template
+INSERT INTO `gameobject_loot_template` (`entry`, `item`, `ChanceOrQuestChance`, `groupid`, `mincountOrRef`, `maxcount`, `condition_id`, `patch_min`, `patch_max`) VALUES

```

Comment:
SQL Error (1062): Duplicate entry '17516-21145' for key 'PRIMARY'

Reply:
Must have been fixed in a later migration, I havent applied.....